clang/include/clang/Basic/TargetOptions.h

```diff
@@ -88,7 +88,7 @@ class TargetOptions {
     COV_5 = 500,
   };
   /// \brief Code object version for AMDGPU.
-  CodeObjectVersionKind CodeObjectVersion = CodeObjectVersionKind::COV_None;
+  CodeObjectVersionKind CodeObjectVersion = CodeObjectVersionKind::COV_5;
```

Comment:
Default was never `none`. It anyway gets overwritten with `MarshallingInfoEnum<TargetOpts<"CodeObjectVersion">,"COV_5">` which was earlier COV4.

This was defined as COV_None in the commit [[NFC] Fix uninitalized member variable use in ASTReader::ParseTargetOptions()](https://github.com/llvm/llvm-project/commit/6f2a865d2f6bc426a61939a0a1acfcb25d5c1a18).

I changed it to cov_5 because it is the new default and leaving it as cov_none might create confusion.